src/status_im/config.cljs

```diff
@@ -144,3 +144,10 @@
 (goog-define STATUS_BACKEND_SERVER_ROOT_DATA_DIR "")
 ;; if you're using android simulator, I suggest set the env variable to "http://10.0.2.2:"
 (goog-define STATUS_BACKEND_SERVER_IMAGE_SERVER_URI_PREFIX "https://localhost:")
+
+;;;; Sentry
+(goog-define SENTRY_DSN "")
```

Comment:
I suggest to use `SENTRY_DSN_STATUS_GO`, because later you'll have separate DSN for `status-mobile` sentry integration, which will be a separate Sentry project.

I've done it like this in desktop:
- `STATUS_BUILD_SENTRY_DSN_STATUS_GO`
- `STATUS_BUILD_SENTRY_DSN_STATUS_DESKTOP`

(you can ignore the `STATUS_BUILD_` prefix)

Reply:
We will go with your naming suggestion @igor-sirotin. Thanks for the suggestion